class.jetpack.php

```diff
@@ -6782,6 +6788,52 @@ public static function build_raw_urls( $url ) {
 		return $url;
 	}
 
+	/**
+	 * Builds an URL using the jetpack.com/redirect service
+	 *
+	 * @param string        $source The URL handler registered in the server
+	 * @param array|string  $args {
+	 *
+	 * 		Additional arguments to build the url
+	 *
+	 * 		@param string $site URL of the current site
+	 * 		@param string $path Additional path to be appended to the URL
+	 * 		@param string $query Query parameters to be added to the URL
+	 * 		@param string $anchor Anchor to be added to the URL
+
+	 * }
+	 *
+	 *
+	 * @return void
+	 */
+	public static function build_redirect_url( $source, $args = array() ) {
+
+		$url = 'https://jetpack.com/redirect';
+
+		$args = wp_parse_args($args);
+		$accepted_args = array( 'site', 'path', 'query', 'anchor' );
+
+		$to_be_added = array(
+			'source' => urlencode( $source )
+		);
+
+		foreach ( $args as $arg_name => $arg_value ) {
+
+			if ( ! in_array( $arg_name, $accepted_args, true ) || empty( $arg_value ) ) {
+				continue;
+			}
+
+			$to_be_added[$arg_name] = urlencode( $arg_value );
```

Comment:
Code format: Missing whitespace in brackets https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#space-usage